lib/smart_proxy_remote_execution_ssh/session.rb

```diff
@@ -121,7 +121,7 @@ def local_command_file(filename)
     end
 
     def remote_command_dir
-      File.join(@remote_working_dir, @command.id)
+      "#{@remote_working_dir}-#{@command.id}"
```

Comment:
Thanks, that makes sense, I'll do that.  That's a bit long, how about `foreman-ssh-cmd-#{@command.id}`?